TestApp/AppDelegate.swift

```diff
@@ -35,7 +35,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         MobileCore.track(state: "Before SDK Init", data: nil)
         MobileCore.setLogLevel(.trace)
         
-        let launchID = ""
```

Comment:
nit: Can you add a comment so that we don't accidentally remove it again in future. 
```
// This is hardcoded for e2e tests. 
let launchID = "94f571f308d5/f986c2be4925/launch-e96cdeaddea9-development"
```